docker/pravega/Dockerfile-java-11

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
Please remove year from copyright statement. It should be:

```
Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
```

Reply:
Done